CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder/Program.cs

```diff
@@ -0,0 +1,136 @@
+using Azure.Communication;
+using Azure.Communication.CallAutomation;
+using Azure.Messaging;
+using CallAutomation_AppointmentReminder;
+using Microsoft.Extensions.FileProviders;
+using static CallAutomation_AppointmentReminder.Logger;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddControllers();
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+var sourceIdentity = await Identity.CreateUser(builder.Configuration["ConnectionString"]);
+var phoneNumberToAddToCall = builder.Configuration["TargetPhoneNumber"];
+builder.Services.AddSingleton(new CallAutomationClient(builder.Configuration["ConnectionString"]));
+builder.Services.AddSingleton(new CallConfiguration(builder.Configuration["ConnectionString"],
+    sourceIdentity.Id,
+    builder.Configuration["ApplicationPhoneNumber"],
+    builder.Configuration["AppBaseUri"]));
+
+var app = builder.Build();
+
+app.MapPost("/api/call", async (CallAutomationClient callAutomationClient, CallConfiguration callConfiguration) =>
```

Comment:
You can inject `ILogger<Program>` here as .NET 6/7 has this already registered in DI for you.